docs/draft/17-url-scheme.md

```diff
@@ -0,0 +1,41 @@
+---
+permalink: /spec/17
+parent: Draft specs
+title: 17/URL SCHEME
+---
+
+# 17/Defined Url Scheme
+
+> Version: 0.1.0
```

Comment:
If my previous suggestions are applied and since my understanding is that our clients are already adhering to this open PR to some extend:

```suggestion
> Version: 0.2.0
```